src/material.cpp

```diff
@@ -39,6 +39,8 @@ namespace {
     {  32,  255,  -3                    }, // Knight      OUR PIECES
     {   0,  104,   4,    0              }, // Bishop
     { -26,   -2,  47,   105,  -149      }, // Rook
+    {   0,    0,   0,     0,     0,   0 }, // Hawk
+    {   0,    0,   0,     0,     0,   0 }, // Elephant
```

Comment:
Please adjust the comments above the QuadraticOurs and QuadraticTheirs array values to take account of the changed order of the columns, and maybe also add the new entries to indicate which parameters are used and to make the triangle complete for aestetic reasons ;-).